build.sbt

```diff
@@ -52,16 +52,34 @@ lazy val publishSettings = Seq(
 )
 
 val circeV = "0.13.0"
+val tcsV = "0.37.0"
+val doobieV = "0.9.0"
+val http4sV = "0.21.6"
 
 lazy val root = (project in file("."))
+  .configs(IntegrationTest)
   .settings(commonSettings)
   .settings(publishSettings)
   .settings(
+    Defaults.itSettings,
     name := "circe-debezium",
     libraryDependencies ++= Seq(
       "io.circe" %% "circe-core" % circeV,
-      "io.circe" %% "circe-parser" % circeV % Test,
-      "io.circe" %% "circe-literal" % circeV % Test,
-      "org.scalatest" %% "scalatest" % "3.1.2" % Test
+      "io.circe" %% "circe-parser" % circeV % "test, it",
+      "io.circe" %% "circe-literal" % circeV % "test, it",
+      "ch.qos.logback" % "logback-classic" % "1.2.3" % "runtime",
```

Comment:
could this be `it` scope?

Reply:
Maybe. I usually leave it as runtime to account for possible transitive dependencies that are also at runtime scope.